test/cmd/policy.sh

```diff
@@ -105,6 +105,7 @@ os::cmd::expect_success_and_text 'oc policy can-i --list --user harold --groups
 
 os::cmd::expect_failure 'oc policy scc-subject-review'
 os::cmd::expect_failure 'oc policy scc-review'
+os::cmd::expect_failure 'oc policy scc-subject-review -u invalid --namespace=noexist'
```

Comment:
The ones above failed on arg problems, but this one is probably failing on something else.  Add a text check.

Reply:
oh, sorry, I think this is just another arg check case, because input file in this command is required.
https://github.com/openshift/origin/blob/717e30862bbe7eb132c57e0e1eda011c9d70def3/pkg/oc/admin/policy/subject_review.go#L82